.circleci/config.yml

```diff
@@ -268,6 +271,123 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: WordPress Android connected tests failed on \`${CIRCLE_BRANCH}\` branch after merge by ${CIRCLE_USERNAME}. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
+  raw-screenshots:
+    executor:
+      name: android/default
+      api-version: "28"
+    working_directory: /tmp/workspace
+    steps:
+      - git/shallow-checkout:
+          init-submodules: true
+      - checkout-submodules
+      - bundle-install/bundle-install:
+          cache_key_prefix: v1-raw-screenshots
+      - android/restore-gradle-cache
+      - copy-gradle-properties
+      - update-gradle-memory
+      - restore-gutenberg-bundle-cache
+      - run:
+          name: Ensure assets folder exists
+          command: mkdir -p libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - attach_workspace:
+          at: libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      # - run:
+      #     name: Build
+      #     environment:
+      #       SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
+      #     command: ./gradlew WordPress:assembleVanillaDebug WordPress:assembleVanillaDebugAndroidTest --stacktrace
+      - run:
+          name: Decrypt credentials
+          command: openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k "${FIREBASE_SECRETS_ENCRYPTION_KEY}"
+      # - android/firebase-test:
+      #     key-file: .circleci/.firebase.secrets.json
+      #     type: instrumentation
+      #     apk-path: WordPress/build/outputs/apk/vanilla/debug/org.wordpress.android-vanilla-debug.apk
+      #     test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+      #     test-targets: package org.wordpress.android.ui.screenshots
+      #     devices: |
+      #       model=blueline,version=28,orientation=portrait
+      #       model=gts3lltevzw,version=28,orientation=default
+      #     # [Trial Limitation] Free plan I use to experiment with Firebase Test Lab has a quota of 10 tests/day
+      #     # locales: ar de_DE en_US es_ES fr_FR iw_IL in it_IT ja_JP ko_KR nl_NL pl_PL pt_BR ru_RU sr sv_SE th tr_TR vi zh_CN zh_TW
+      #     locales: en_US in
+      #     project: wpandroid-74b6c
```

Comment:
⚠️ This project ID will need to be replaced by `api-project-108380595987` once integrated with a8c's Firebase instance